apps/dashboard/app/(app)/settings/root-keys/[keyId]/permissions/permission-manager-card.tsx

```diff
@@ -0,0 +1,133 @@
+
+  // TODO: Add the ability to act like an accordion, initially collapsed (prop should be called `expandable`)
+};
+
+// TODO: Add a visualization for when there's no active permissions
+export function PermissionManagerCard(props: PermissionManagerCardProps) {
+  const activePermissions = Object.entries(props.permissionsStructure).filter(
+    ([_category, allPermissions]) => {
+      const amountActiveRules = Object.entries(allPermissions).filter(
+        ([_action, { description: _description, permission }]) => {
+          return props.permissions.some((p) => p.name === permission);
+        },
+      );
+
+      return amountActiveRules.length > 0;
+    },
+  );
+
+  return (
+    <Card>
+      <CardHeader>
+        <div className="flex items-center">
+          <CardTitle className="flex grow shrink-0">{props.permissionManagerTitle}</CardTitle>
+          <Dialog>
+            <DialogTrigger asChild>
+              <Button className="flex grow-0 shrink-0" variant="outline">
+                Edit
+              </Button>
+            </DialogTrigger>
+
+            <DialogContent className="sm:max-w-[640px] max-h-[70vh] overflow-y-scroll">
+              <DialogHeader>
+                <DialogTitle>{props.permissionManagerTitle}</DialogTitle>
+                <DialogDescription>{props.permissionManagerDescription}</DialogDescription>
+              </DialogHeader>
+
+              <div className="flex flex-col w-full gap-4">
+                {Object.entries(props.permissionsStructure).map(([category, allPermissions]) => (
+                  <div className="flex flex-col gap-2">
+                    <span className="font-medium">{category}</span>{" "}
+                    <div className="flex flex-col gap-1">
+                      {Object.entries(allPermissions).map(
+                        ([action, { description, permission }]) => {
+                          return (
+                            <PermissionToggle
+                              key={action}
+                              rootKeyId={props.keyId}
+                              permissionName={permission}
+                              label={action}
+                              description={description}
+                              checked={props.permissions.some((p) => p.name === permission)}
+                            />
+                          );
+                        },
+                      )}
+                    </div>
+                  </div>
+                ))}
+              </div>
+            </DialogContent>
+          </Dialog>
+        </div>
+        <CardDescription>{props.permissionManagerDescription}</CardDescription>
+      </CardHeader>
+      <CardContent className="text-sm">
+        <div className="flex flex-col gap-4">
+          {activePermissions.length === 0 && (
+            <p className="text-content-subtle">
+              There are no active rules. To get started, edit the permissions.
```

Comment:
I think we shouldn't introduce a new word ("rules") when talking about permissions. Cause now it sounds like rules and permissions might be different things.

There are also some variables referencing `rule`, can you edit those too?